zeppelin-server/src/test/java/org/apache/zeppelin/rest/NotebookRestApiTest.java

```diff
@@ -107,14 +108,24 @@ public void testRunParagraphJob() throws IOException {
     assertEquals(p.getStatus(), Job.Status.FINISHED);
 
     // run non-blank paragraph
-    p.setText("test");
+    p.setText("%spark.pyspark\nprint(\"test\")");
     post = httpPost("/notebook/job/" + note1.getId() + "/" + p.getId(), "");
     assertThat(post, isAllowed());
     resp = gson.fromJson(post.getResponseBodyAsString(),
             new TypeToken<Map<String, Object>>() {}.getType());
     assertEquals(resp.get("status"), "OK");
     post.releaseConnection();
-    assertNotEquals(p.getStatus(), Job.Status.READY);
+    long startTimeInMins = (System.currentTimeMillis() / (1000 * 60)) % 60;
```

Comment:
You’d better move this logic under the test of `notebookservice`

Reply:
`NotebookService.runParagraph` is already tested in" NotebookServiceTest.testParagraphOperations ", so I decided to test exactly the REST API method, because even if ` NotebookService.runParagraph` works, the REST API method could fail because of wrong argument preprocessing for `NotebookService.runParagraph`.

Reply:
@jongyoul Do you agree with me?

Reply:
Sorry for the late response. BTW, if this is already tested in `NotebookServiceTest`, it would be ok to check all parameters were passed exactly with mock object. WDYT? Personally, I like this kind of conservative code but at the same time, I should consider the flakiness of all tests. This assumes one minute but it could toke more time with some reason which is not related to actual failure.

Reply:
@jongyoul I agree with you, so I reverted this changes, maybe I'll refactor them later using PowerMock.